### PR TITLE
add aojea to OWNERS

### PR DESCRIPTION
**Pull request by aojea**
as Sig testing TL

### CODE REVIEW

File: OWNERS
Comment (upodroid) [inline OWNERS:4]: Can you move `MushuEE` and `spiffxp` to emeritus from reviewers and approvers? Also, can you add me as an approver? I can merge this PR manually once its ready

Comment (aojea) [reply OWNERS:4]: done

**Review by upodroid — APPROVED** [review]

**Comment by BenTheElder** [comment]
thank you both!

**Pull request merged**
